// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// A runtime sample of a task's resource demands.

package firmament;

message MachinePerfStatisticsSample {
  required string resource_id = 1;
  required uint64 timestamp = 2;
}
